board/stm32f4/peripherals.h

```diff
@@ -9,6 +9,7 @@ static void gpio_usb_init(void) {
   GPIOA->OSPEEDR = GPIO_OSPEEDER_OSPEEDR11 | GPIO_OSPEEDER_OSPEEDR12;
 }
 
+#if defined(ENABLE_SPI)
```

Comment:
```suggestion
#ifdef ENABLE_SPI
```

after that it should be good to merge